pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -2545,7 +2552,7 @@ func TestContextValid(t *testing.T) {
 	}
 }
 
-func TestContextInvalid(t *testing.T) {
+func TestContextInvalidMatrix(t *testing.T) {
```

Comment:
this test is for invalid contexts everywhere, please don't remove the existing tests for context variables in non-matrix fields